react/features/filmstrip/components/native/TileView.js

```diff
@@ -145,7 +146,7 @@ class TileView extends Component<Props> {
             return 2;
         }
 
-        return Math.min(3, participantCount);
+        return Math.min(3, participantCount);*/
```

Comment:
Why not just remove the code instead of commenting it out? If we need to get it back, it's in git history.